### PR TITLE
load-test client

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Looking to merge the load-test client into the main branch. Notably I've implemented a couple of additions which will be needed in the upcoming performance tests:

- Concurrent calls support
- Screen sharing support

There's some work still needed to polish (e.g. proper config) it but I feel it's a good time to include it, if anything to avoid constantly switching between branches.

### CODE REVIEW

File: lt/client.go [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+		}
+		defer file.Close()
+
+		// Open on oggfile in non-checksum mode.
+		ogg, _, oggErr := oggreader.NewWith(file)
+		if oggErr != nil {
+			log.Fatalf(oggErr.Error())
+		}
+
+		<-time.After(2 * time.Second)
```
Comment (streamer45) [inline lt/client.go:185]: @cpoile Could you remind me why we have to sleep for a bit here? I've kept the semantic you introduced but forgot what we were trying to achieve (or avoid that is).

**Review by cpoile — APPROVED** [review]
👍

**Comment by streamer45** [comment]
I had to revert one small detail as setting up the audio track is actually required even if clients are muted.

**Comment by streamer45** [comment]
@cpoile Will finally merge this so that we can update the documentation and link to the load-test docs as I've already heard a couple of customers in need for something to test Calls performance on their instances.

This should also unblock any further improvement such as the simulative logic you were working on.

**Pull request merged**